CMP1903_A1_2324/Game.cs

```diff
@@ -15,7 +15,25 @@ internal class Game
          * rolls could be continous, and the totals and other statistics could be summarised for example.
          */
 
-        //Methods
+        /// <summary>
+        /// The "Game" class contains the code relevant for generating the die game, in this case that is the instantiation and rolling of the three dies
+        /// along with summing of their values. The method "StartGame" which is responsible for rolling the dies and summing them returns the die values and their sum.
+        /// </summary> 
 
+        Die die1 = new Die(); //Dies are instantiated here.
```

Comment:
This in-line comment can be turned into a header comment to describe the code in sections.